src/wallet/walletutil.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
Are you Satoshi? :slightly_smiling_face: If not, I guess you can remove this copyright line (and the same from `walletutil.h`)